doc/03-Configuration.md

```diff
@@ -11,7 +11,8 @@ Configuration of the Redis that Icinga writes to.
 
 Option                   | Description
 -------------------------|-----------------------------------------------
-address                  | **Required.** Redis host:port address or absolute Unix socket path.
+host                     | **Required.** Redis host or absolute Unix socket path.
+port                     | **Optional.** Redis port. Defaults to `6379`. Specify `6380` when using the `icingadb-redis` package.
```

Comment:
Should be "if" instead of "when", [I guess](https://dictionary.cambridge.org/grammar/british-grammar/if-or-when).